service/urls.py

```diff
@@ -23,4 +23,6 @@
     url(r'^posts/(?P<pk>[0-9a-z\\-]+)/', service.posts.views.AllPostsViewSet.as_view({"get": "retrieve"}),
         name='post-detail'),
     url(r'^author/(?P<pk>[0-9a-z\\-]+)/posts/', service.posts.views.AuthorPostsList.as_view(), name='author-posts-list'),
+    url(r'^author/(?P<me>[0-9a-z\\-]+)/friends/(?P<you>[0-9a-z\\-]+)',
```

Comment:
Actually, I'm wrong. In the case where we don't know about the other author, we still need to return its URI in the response, so the other host name is actually required, at least in that case.